src/Microsoft.TestPlatform.CrossPlatEngine/Microsoft.TestPlatform.CrossPlatEngine - Copy.csproj

```diff
@@ -0,0 +1,72 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why there is CrossPlatform - Copy project?

Reply:
It's a mistake when I was copy pasting the empty PublicAPI files. Removing.